chain/actors/builtin/miner/diff.go

```diff
@@ -117,11 +209,69 @@ func (m *sectorDiffer) Modify(key uint64, from, to *cbg.Deferred) error {
 	return nil
 }
 
-func (m *sectorDiffer) Remove(key uint64, val *cbg.Deferred) error {
+func (m *sectorDiffContainer) Remove(key uint64, val *cbg.Deferred) error {
 	si, err := m.pre.decodeSectorOnChainInfo(val)
 	if err != nil {
 		return err
 	}
 	m.Results.Removed = append(m.Results.Removed, si)
 	return nil
 }
+
+func SectorsAmtBitwidth(c cid.Cid) (int, error) {
```

Comment:
Can we generate this and similar functions for other actors?

Reply:
We can if specs-actors remains consistent in their usage of constants for defining different AMT Bitwidths. Would you prefer this method to be generated on the miner shim instead? (And similarly for the market actor?)

Reply:
That would be cool I think

Reply:
Done.